.github/workflows/publish.yml

```diff
@@ -20,21 +20,21 @@ jobs:
         os: [ubuntu-20.04, ubuntu-latest]
 
     steps:
-    - uses: actions/checkout@v3
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v3
-      with:
-        node-version: ${{ matrix.node-version }}
-        cache: 'npm'
-    - run: npm install
-    - run: npm run build --if-present
-    - run: npm run prettier
-    - name: Release
-      if: github.ref == 'refs/heads/main' && '${{ matrix.node-version }}' == '18.x' && '${{ matrix.os }}' == 'ubuntu-latest'
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-      run: |
-        git config --global user.name "Github Actions"
-        git config --global user.email "laryssa.abdala@kitware.com"
-        npm run semantic-release
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+      - run: npm install
+      - run: npm run build --if-present
+      - run: npm run prettier
+      - name: Release
+        if: matrix.node-version == '18.x' && matrix.os == 'ubuntu-latest'
```

Comment:
We should still have the check for the `main` branch here (do not try to run on pull requests).

Reply:
Looks like semantic-release does the right thing without the branch check:
`[6:27:40 PM] [semantic-release] › ℹ  This run was triggered by a pull request and therefore a new version won't be published.`

Plus might work when tagging?

Reply:
Ok, let's forward to semantic-release.